### PR TITLE
Add mouse scroll wheel mapping

### PR DESCRIPTION
**Pull request by xxv**
This scrolls with the natural scrolling style, moving with the movement
of the page, not the scrollbar.

### CODE REVIEW

**Review by algernon — COMMENTED** [review]
While I agree that having a scroll wheel on the factory firmware would be awesome, doing so has a few implications, like having to update the printed & laminated sheet too, as that includes mouse keys as well. Not doing that may end up surprising new users.

Another thing I'm unsure about is the order of up / down scroll. It seems that scrolling down is by the "mouse up" key. That seems counter-intuitive to me.

In short, I'm a bit torn here. On one hand, scroll wheel is <3. On another, the order, and the implications make this change less practical.

But it's up to @obra in the end.

**Comment by obra** [comment]
I think we want to create a second model 01 firmware demo repo with a 'showy' firmware that has examples of all the bells and whistles.
